docs/lib-documentation.md

```diff
@@ -41,6 +40,19 @@ Gets the [Service][service] interface representing the service with the given ID
 
 The [Service][service] implementation representing the service.
 
+### getServiceContext(ServiceID serviceId, [ContainerConfigFactory\<S\>][containerconfigfactory] configFactory)
```

Comment:
I see why you needed the containerConfigFactory here, though I think we'll want to think about how we can not need that here - e.g. maybe by storing the information on the API container's side

Reply:
We will change this in the next PR

Reply:
I removed the containerConfigFactory argument from here.